counterexamples/transportation/segment/road/restrictions/prohibited_transitions/bad-sequence-empty.yaml

```diff
@@ -0,0 +1,22 @@
+---
+id: overture:transportation:counterexample:bad-sequence-duplicate-entry
+type: Feature
+geometry:
+  type: LineString
+  coordinates: [[0, 0], [1, 1]]
+properties:
+  theme: transportation
+  type: segment
+  version: 1
+  update_time: "2024-03-06T09:44:46-07:00"
+  subtype: road
+  connector_ids:
+    - overture:transportation:example:connector:1
+  road:
+    class: primary
+    restrictions:
+      prohibited_transitions:
+        - sequence: []
+          final_heading: forward
+  ext_expected_errors:
+    - "sequence/minItems] minimum 1 items required, but found 0 items"
```

Comment:
Opening bracket [ missing

Reply:
Thanks! I actually did that on purpose, because everything left of `sequence/minItems` is pretty verbose and not worth matching against...